drivers/storage/digitalocean/storage/digitalocean_storage.go

```diff
@@ -0,0 +1,336 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_digitalocean
+
+package storage
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/digitalocean/godo"
+
+	gofig "github.com/akutz/gofig/types"
```

Comment:
Hi @protochron,

I won't repeat this comment in other files, but it's relevant to them. I prefer import blocks to be like so:

```go
import (
	stdlib

	external_pkgs

	external_pkgs_relevant_to_storage_driver

	project_pkgs

	project_pkgs_relevant_to_storage_driver
```

For this file's imports I'd organize them like so:

```go
import (
	"fmt"
	"path/filepath"
	"strconv"
	"time"

	gofig "github.com/akutz/gofig/types"
	"github.com/akutz/goof"
	log "github.com/Sirupsen/logrus"

	"github.com/digitalocean/godo"

	"github.com/codedellemc/libstorage/api/context"
	"github.com/codedellemc/libstorage/api/registry"
	"github.com/codedellemc/libstorage/api/types"

	do "github.com/codedellemc/libstorage/drivers/storage/digitalocean"
	doUtils "github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
)
```

Not all my own sources follow this as I've developed this preference over time, but I do try to update files as I work on them. It just makes it easier to scan the imports when looking for specific types of packages.